torch/fx/proxy.py

```diff
@@ -42,7 +42,13 @@ def create_node(self, kind : str, target : Target,
         if kind == 'call_function' and self.check_mutable_operations:
             check_for_mutable_operation(target, args, kwargs)
 
-        return self.graph.create_node(kind, target, args, kwargs, name, type_expr)
+        node = self.graph.create_node(kind, target, args, kwargs, name, type_expr)
```

Comment:
Or shall I only apply this behavior to PythonKeyTracer? 
Do we want PythonKeyTracer to be the parents class for all retracing retracer ?

Reply:
I think we can probably put this logic inside of ProxyTensor itself, right?

Reply:
I am not sure how to do it... or if it's possible? 

To propagate stack trace from one FX node to another FX node in the new graph, I am reading stack trace from an Interpreter.current_node, and writing stack trace to a new node being created by a Tracer. 

IIUC, ProxyTensor and ProxyTorchDispatchMode doesn't carry such stack_trace info...